templates/includes/nav.haml

```diff
@@ -7,51 +7,146 @@
       %a.icon-nav-messages{href:"{% url 'msgs.msg_inbox' %}", class:"{% active request 'inbox|outbox|broadcast|call|msg/filter|msg/flow|msg/archived|failed' %}"}
         .title
           -trans "messages"
+      %div.custom-tooltip
+        %div.custom-tooltip-arrow
+        %span.custom-tooltip-icon
+          %div.around
+            %i.icon-nav-messages-inverted
+        %span.custom-tooltip-title
+          -trans "Messages"
+        %div.custom-tooltip-separator
+        %span.custom-tooltip-description
+          -trans "Manage your inbox of inbound in outbound message"
```

Comment:
@srugano - This string should be "Manage your inbox of inbound **and** outbound messages", take a look on Wrike ticket to see the descriptions that Nelson added for each menu item: https://www.wrike.com/open.htm?id=390762050

Reply:
Updated the texts of [tooltips](https://github.com/greatnonprofits-nfp/rapidpro/commit/0566302c58f91066846183ec2d8a1fea41901a4e)